addon/serializer.js

```diff
@@ -23,6 +23,34 @@ export default DS.JSONSerializer.extend(DS.EmbeddedRecordsMixin, {
     this._super(hash, typeClass, snapshot, options);
   },
 
+  serializeAttribute(snapshot, json, key, attribute) {
+    let type = attribute.type;
+
+    if (this._canSerialize(key)) {
+      let value = snapshot.attr(key);
+      if (type) {
+        if (type === 'string') {
+          if (!Ember.isNone(value)) {
+            value = value.replace(/\"/g, '\\"');
```

Comment:
how come we are double escaping here?

also, could we possibly replace with `JSON.stringify`?